### ISSUE TITLE
prefix_module incorrectly identifies an interface instance

### ISSUE THREAD

**Issue by SnakePin** [dalance/svlint#154]
As the title says, the `prefix_module` rule incorrectly identifies an interface instantiation as a module instantiation and it results in an incorrect warning produced by the rule.
I know it's not trivial to identify an interface instantiation without finding the interface definition but I'll open an issue regardless.
I thought that the issue was more suitable for this project as opposed to sv-parser as modifying the rule seems like an easier way.
Example:
```systemverilog
ifc_interface interface_instance(); //`module` name must have prefix "mod_"svls(prefix_module)
```

**Comment by DaveMcEwan** [comment]
I've submitted PR #155 to only perform the prefix check on module declarations (but not instantiations).
@SnakePin Do you agree that this is a suitable solution?
